client/src/components/Pill/Pill.stories.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Pill from "./Pill";
+
+storiesOf("Pill", module).add("Default", () => <Pill />);
```

Comment:
We need a way to pass down the color and the text as a prop. Can you implement that?